assets/js/modules/reader-revenue-manager/components/dashboard/ReaderRevenueManagerSetupCTABanner.js

```diff
@@ -37,268 +35,164 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { useDispatch, useSelect } from 'googlesitekit-data';
-import { Button } from 'googlesitekit-components';
 import {
 	BREAKPOINT_SMALL,
 	BREAKPOINT_TABLET,
 	useBreakpoint,
 } from '../../../../hooks/useBreakpoint';
 import useActivateModuleCallback from '../../../../hooks/useActivateModuleCallback';
-import whenInactive from '../../../../util/when-inactive';
-import { withWidgetComponentProps } from '../../../../googlesitekit/widgets/util';
-import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
-import {
-	READER_REVENUE_MANAGER_MODULE_SLUG,
-	READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY,
```

Comment:
@aaemnnosttv 
1. Should I be creating a migration file to update the dismissed prompts key in the DB for users that may have already dismissed the banner? In the new code, the notification ID is used as the dismissal key.
2. Or should we change the notification ID here to be the same as the original dismissal key - but this would be 'hacky' as the ID would have a funny name here. And the GA events will now be tracked to a different ID than the original ID used.
3. Should we keep things as it is and have some of the users who have RRM enabled dismiss the banner again?

Reply:
I think keeping as-is is the best option here but if the RRM setup prompt is retryable, then I'd be a bit concerned if folks already dismissed it multiple times and now they'd need to dismiss it multiple times again. Is that the case? It looks like yes.

In that case, I think we can simply check for the presence of the old dismissal in the RRM notification's `checkRequirements` function. That way, everything else runs as usual but we still account for an existing dismissal.

Reply:
It sounds like a good easy hack which we can remove if we do any migration for dismissed-items or prompts in the future.

Reply:
I wouldn't say it's a hack 😄